core/src/test/scala/org/apache/spark/sql/delta/expressions/RangePartitionIdSuite.scala

```diff
@@ -0,0 +1,83 @@
+/*
+ * Copyright (2021) The Delta Lake Project Authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.spark.sql.delta.expressions
+
+import scala.reflect.ClassTag
+
+import org.apache.spark.{Partitioner, RangePartitioner, SparkFunSuite}
+import org.apache.spark.sql.catalyst.expressions._
+import org.apache.spark.sql.test.SharedSparkSession
+
+
+class RangePartitionIdSuite
+  extends SparkFunSuite with ExpressionEvalHelper with SharedSparkSession {
```

Comment:
`ExpressionEvalHelper` [extends](https://github.com/apache/spark/blob/branch-3.2/sql/catalyst/src/test/scala/org/apache/spark/sql/catalyst/expressions/ExpressionEvalHelper.scala#L48) `ScalaCheckDrivenPropertyChecks` which is part of `"org.scalatestplus" %% "scalacheck-1-15"`